src/components/Player/Player.jsx

```diff
@@ -14,19 +16,20 @@ import {
   HeroesPage,
   ProsPage,
   RankingsPage,
+  HistogramsPage,
   RecordsPage,
   CountsPage,
 } from './Pages';
-import TabBar from '../TabBar';
 import { playerPages } from '../Header/Pages';
 
-const playerPagesMapped = (accountId) => playerPages.map(page => ({
-  ...page,
-  route: `/players/${accountId}/${page.name.toLowerCase()}`,
-  label: page.name,
+const playerPagesMapped = (accountId) => playerPages.map(({ name, extra = '', ...rest }) => ({
+  ...rest,
+  route: `/players/${accountId}/${name.toLowerCase()}${extra ? `/${extra}` : ''}`,
+  label: name,
+  extra,
```

Comment:
this "extra" param is not needed if histogram components can access route state?